types/react-notifications-component/index.d.ts

```diff
@@ -0,0 +1,56 @@
+// Type definitions for react-notifications-component 2.4
+// Project: https://www.npmjs.com/package/react-notifications-component
```

Comment:
This needs to be https://github.com/teodosii/react-notifications-component

Reply:
Ok, got it. I marked this as resolve. Is there anything further I need to do?